docs/reference/api/connectivity/networksocket/EthInterface.md

```diff
@@ -1,19 +1,45 @@
 ## Ethernet
 
-The EthInterface provides a C++ API for connecting to the internet over Ethernet.
+The `EthInterface` provides a C++ API for connecting to the internet over Ethernet. By default, this class does not require any configuration. It is able to pick up the default Ethernet driver for the target and select the correct network stack.
```

Comment:
Is there anything else developers should know about what this API is and what it does? This section seems short.

Reply:
Its hard to see what should be inserted here.

Networking architecture in general is described in #470 and Ethernet MAC drivers will be explained on their [own page](https://github.com/ARMmbed/mbed-os/blob/feature-emac/features/netsocket/emac-drivers/README.md) (to be moved to Handbook later on).

This whole API usage is concised in the 2 line example below in the Usage section.

```
EthernetInterface eth;
eth.connect();
```